src/ip-payment/test/logic.feature

```diff
@@ -6,6 +6,7 @@ Feature: IP Payment Contract
 
   Scenario: Payment of a specified amount should be made
     When the current time is "2019-03-04T16:34:00-05:00"
+    And the UTC offset is -5
```

Comment:
This is neat!

Reply:
I think it works. Reminds me I need to add that to the CLIs too.